doc/cygnus-ngsi-ld/user_and_programmer_guide/introduction.md

```diff
@@ -1,8 +1,13 @@
-# <name="top"></a>User and programmer guide
+#User and programmer guide
 This document describes how to use Cygnus once it has been [installed](../installation_and_administration_guide/introduction.md). It describes how to program extensions for it as well.
 
 ## Intended audience
-This documenttion is intended to be used by developers, if you want to only use this cygnus-ngsi-ld pleasego
-to the [Asdministration section](../installation_and_administration_guide/introduction.md). It describes how to program extensions for it as well.
-
-
+This documenttion is intended to be used by developers, if you want to only use this cygnus-ngsi-ld please go
+to the [Administration section](../installation_and_administration_guide/introduction.md). It describes how to program extensions for it as well.
+## Structure of the document
+* [Introduction](./introduction.md)
```

Comment:
This is a weird self-linking of the same document... I'd suggest to remove this entry.

Reply:
Done